docs/AboutUs.md

```diff
@@ -19,15 +19,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Team Lead / Developer
 * Responsibilities: Deliverables and Scheduling
 
-### Jane Doe
+### Chia Min Jiun
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/chiaminjiun.png" width="200px">
```

Comment:
The filename of the profile photo needs to be in the format <github_username_in_lowercase>.png for grading purposes.

Reply:
filename changed